change/@microsoft-teams-js-286cc515-1cef-4d54-9af6-d869e00d96c9.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "Updated `copilot.eligibility.getEligibilityInfo` to get the eligibility info from the host if it is not already available.",
```

Comment:
API consumers might be more interested in the changing function definition from sync to async